src/resources/s3.js

```diff
@@ -77,6 +77,29 @@ class S3Bucket { // eslint-disable-line padded-blocks
 		};
 	}
 
+	/**
+	 * Translate the logging configuration into the 'logging params' for the AWS SDK.
+	 *
+	 * @param {string} bucketName the bucket name
+	 * @param {PublicAccessBlockConfiguration} publicAccessBlockConfiguration Public Access Block configuration
+	 * @returns {Object} the parameters for `putPublicAccessBlock`
+	 */
+	_translatePublicAccessBlockConfiguration(bucketName, publicAccessBlockConfiguration) {
+		let status;
+		if (!publicAccessBlockConfiguration) {
+			status = {};
+		} else {
+			status = {
+				BlockPublicAcls: publicAccessBlockConfiguration.blockPublicAcls,
```

Comment:
Could you also add all other properties of https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-publicaccessblockconfiguration.html please?

Reply:
Fixed by https://github.com/Collaborne/kubernetes-aws-resource-service/pull/43/commits/93bd84280243aff0ac79991da6a749b97034808c